spec/rubocop/cop/rspec/repeated_example_group_description_spec.rb

```diff
@@ -240,4 +240,180 @@
       end
     RUBY
   end
+
+  it 'does not register offense for example group with skip' do
```

Comment:
This description is not exactly fair, is it?
We do not register an offence, but because context descriptions are different. Wouldn’t this pass before the fix, too?